python/cudf/cudf/tests/test_binops.py

```diff
@@ -2166,7 +2071,7 @@ def test_binops_with_NA_consistent(dtype, op):
         ),
         (
             operator.truediv,
-            20,
```

Comment:
This scalar value will have a different inferred type than the next `Decimal128Dtype` below since we are removing testing with `cudf.Scalar`, so replacing with an "equivalent" test data of `["20", "20"]`